1) The marks awarded for an assignment set for a Year 8 class of 20 students were as
follows:
6 7 5 7 7 8 7 6 9 7 4 10 6 8 8 9 5 6 4 8

In [19]:
import numpy as np
from collections import Counter

In [20]:
a = [6,7,5,7,7,8,7,6,9,7,4,10,6,8,8,9,5,6,4,8]

In [21]:
np.mean(a)

6.85

In [22]:
np.median(a)

7.0

In [12]:
data = Counter(a)

In [14]:
data.most_common()

[(7, 5), (6, 4), (8, 4), (5, 2), (9, 2), (4, 2), (10, 1)]

Hence, mode is 7 since it occured 5 times.

In [53]:
np.std(a)

1.5898113095584647

2) The number of calls from motorists per day for roadside service was recorded for a
particular month:
28, 122, 217, 130, 120, 86, 80, 90, 140, 120, 70, 40, 145, 113, 90, 68, 174, 194, 170,
100, 75, 104, 97, 75,
123, 100, 75, 104, 97, 75, 123, 100, 89, 120, 109

In [55]:
val = [28, 122, 217, 130, 120, 86, 80, 90, 140, 120, 70, 40, 145, 113, 90, 68, 174, 194, 170, 100, 75, 104, 97, 75,123, 100, 75, 104, 97, 75, 123, 100, 89, 120, 109]

In [57]:
np.mean(val)

107.51428571428572

In [63]:
val.sort()

In [58]:
np.median(val)

100.0

In [64]:
data = Counter(val)

In [66]:
data.most_common()

[(75, 4),
 (100, 3),
 (120, 3),
 (90, 2),
 (97, 2),
 (104, 2),
 (123, 2),
 (28, 1),
 (40, 1),
 (68, 1),
 (70, 1),
 (80, 1),
 (86, 1),
 (89, 1),
 (109, 1),
 (113, 1),
 (122, 1),
 (130, 1),
 (140, 1),
 (145, 1),
 (170, 1),
 (174, 1),
 (194, 1),
 (217, 1)]

Hence, Mode is 75 since it occured highest times

In [67]:
np.std(val)

38.77287080168403

3) The number of times I go to the gym in weekdays, are given below along with its
associated probability:

x = 0, 1, 2, 3, 4, 5
f(x) = 0.09, 0.15, 0.40, 0.25, 0.10, 0.01

Calculate the mean no. of workouts in a week. Also evaluate the variance involved in
it.

In [23]:
x= [0, 1, 2, 3, 4, 5]
y = [ 0.09, 0.15, 0.40, 0.25, 0.10, 0.01]

In [24]:
v = [x*y for x,y in zip(list(x),list(y))]

In [25]:
sum(v)

2.15

In [28]:
z = [((x-sum(v))**2 *y)for x,y in zip(list(x),list(y))]
sum(z)

1.2275

Hence the mean is 2.15 and the variance is 1.2275

Q4) Let the continuous random variable D denote the diameter of the hole drilled in an
aluminum sheet. The target diameter to be achieved is 12.5mm. Random
disturbances in the process often result in inaccuracy.
Historical data shows that the distribution of D can be modelled by the PDF (d) =
20e−20(d−12.5), d ≥ 12.5. If a part with diameter > 12.6 mm needs to be scrapped,
what is the proportion of those parts? What is the CDF when the diameter is of 11
mm? What is your conclusion regarding the proportion of scraps?

Q5) A company manufactures LED bulbs with a faulty rate of 30%. If I randomly select 6
chosen LEDs, what is the probability of having 2 faulty LEDs in my sample?
Calculate the average value of this process. Also evaluate the standard deviation
associated with it.

In [6]:
from math import e

In [15]:
((e**1.2)*1.8**)

92.51892924363653

Q6) Gaurav and Barakha are both preparing for entrance exams. Gaurav attempts to
solve 8 questions per day with a correction rate of 75%, while Barakha averages
around 12 questions per day with a correction rate of 45%. What is the probability
that each of them will solve 5 questions correctly? What happens in cases of 4 and 6
correct solutions? What do you infer from it? What are the two main governing
factors affecting their ability to solve questions correctly? Give a pictorial
representation of the same to validate your answer.

In [16]:
#variance in the case of Gaurav
8*.75

6.0

In [17]:
# Variance in the case of Barakha
12*.45

5.4

In [40]:
#probality of Gaurav get 5 questions correctly

a= 16

In [41]:
#probality of Barakha to get 5 questions correctly

b = 17

In [42]:
#probability that each of them will solve 5 questions correctly
(a+b)/2

16.5

In [43]:
#In the case of getting 4 correct answer
(13+16)/2

14.5

In [44]:
#In the case of getting 6 correct answer
(16+15.5)/2

15.75

Q7) Customers arrive at a rate of 72 per hour to my shop. What is the probability of k
customers arriving in 4 minutes? a) 5 customers, b) not more than 3 customers, c)
more than 3 customers. Give a pictorial representation of the same to validate your
answer.

In [46]:
# in 4 mins customers coming are
a = (72/60)*4
a

4.8

a) probability of 5 customers arriving in 4 minutes is 17%
b) not more than 3 customers is 28.3%
c) more than 3 customers is 17%

Q8) I work as a data analyst in Aeon Learning Pvt. Ltd. After analyzing data, I make
reports, where I have the efficiency of entering 77 words per minute with 6 errors per
hour. What is the probability that I will commit 2 errors in a 455-word financial report?
What happens when the no. of words increases/decreases (in case of 1000 words,
255 words)?
How is the λ affected?
How does it influence the PMF?
Give a pictorial representation of the same to validate your answer.

Answer

a) probability that I will commit 2 errors in a 455-word financial report is 9%
b) 1.23 and 0.32 error in 1000 and 255 words and as words increases λ also increases.
c) PMF will increase.

Q 10) Please compute the following:
a) P(Z > 1.26), . P(Z < −0.86), P(Z > −1.37), P(−1.25 < Z < 0.37), . P(Z ≤ −4.6)
b) Find the value z such that P(Z > z) = 0.05
c) Find the value of z such that P(−z < Z < z) = 0.99

Answer

Section a) 

P(Z > 1.26)= 1-P(Z < 1.26)= .1038,
P(Z < −0.86) = .1949,       
P(Z > −1.37) =  1-P(Z < −1.37) = .9147,
P(−1.25 < Z < 0.37) = p(z<0.37)-p(z<-1.25) = .5387

Section b)

P(Z > z) = 0.05 here z = 1.65

Section c)

P(−z < Z < z) = 0.99, here z = 3.4

Q11) The current flow in a copper wire follow a normal distribution with a mean of 10 mA
and a variance of 4 (mA)2.
What is the probability that a current measurement will exceed 13 mA? What is the
probability that a current measurement is between 9 and 11mA? Determine the
current measurement which has a probability of 0.98.

Answer

ection a) 

p(x>13mA) = 1-.933 = .67 or 67%

Section b)

P(9<x<11) = p(x<11)-p(x<9) = .382 or 38.2%

Section c)

14.17

Q12) The shaft in a piston has its diameter normally distributed with a mean of 0.2508 inch
and a standard deviation of 0.0005 inch. The specifications of the shaft are 0.2500 ∓
0.0015 inch. What proportion of shafts are in sync with the specifications? If the
process is centered so that the mean is equal to the target value of 0.2500, what
proportion of shafts conform to the new specifications? What is your conclusion from
this experiment?

Answer

case 1)
Here we have to calculate p(.2485)<=x(a)=<p(.2515) and p(.2515) = 0.919 , p(.2485) = 0, hence p(.2515) - p(.2485) = 0.919

case 2)
p(.2515) - p(.2485) = 0.999 - 0.0001 = 0.998